util/src/main/scala/com/scalableminds/util/geometry/BoundingBox.scala

```diff
@@ -47,31 +47,27 @@ object BoundingBox {
 
   import play.api.libs.json._
 
-  val formRx = "\\s*([0-9]+),\\s*([0-9]+),\\s*([0-9]+)\\s*,\\s*([0-9]+),\\s*([0-9]+),\\s*([0-9]+)\\s*".r
+  private val formRx = "\\s*([0-9]+),\\s*([0-9]+),\\s*([0-9]+)\\s*,\\s*([0-9]+),\\s*([0-9]+),\\s*([0-9]+)\\s*".r
 
-  def empty =
+  def empty: BoundingBox =
     BoundingBox(Point3D(0, 0, 0), 0, 0, 0)
 
-  def toForm(b: BoundingBox): Some[String] =
-    Some(
-      "%d, %d, %d, %d, %d, %d".format(
-        b.topLeft.x,
-        b.topLeft.y,
-        b.topLeft.z,
-        b.topLeft.x + b.width,
-        b.topLeft.y + b.height,
-        b.topLeft.z + b.depth
-      ))
-
-  def fromForm(s: String): Box[BoundingBox] =
+  def createFrom(s: String): Box[BoundingBox] =
```

Comment:
Is `createFrom` an established term for constructor-like methods? Couldn't it overload the constructor instead? (Probably not, because of the `Box`?). I found `fromForm` or maybe `fromFormString` quite speaking (also, since it would be similar in python-land), but `createFrom` is fine as-well.

Reply:
Yes, overwriting the constructor is not possible as you described. fromX would have been fine by me, but there were already a number of createFrom methods here with different parameters and I didn’t want to make the diff all across wK large by renaming them all now.